src/main/java/org/kcctl/command/PatchConnectorCommand.java

```diff
@@ -49,7 +49,7 @@ public class PatchConnectorCommand implements Callable<Integer> {
     CommandSpec commandSpec;
 
     @Parameters(paramLabel = "CONNECTOR NAME", description = "Name of the connector", completionCandidates = ConnectorNameCompletions.class)
-    Set<String> names;
+    Set<String> names = Set.of();
```

Comment:
Is this change necessary? All of the tests pass without it.

Reply:
Avoid getting NPE when you're not specified the names of connector.